src/BirdMessenger/Builder/ITusHttpClientBuilder.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace BirdMessenger.Builder
+{
+    public interface ITusHttpClientBuilder<TService>
```

Comment:
it should rename ITusHttpClientConfigre<TService>  ？

Reply:
> ok，I merged, Do you have email? I want to send message to you

I updated my profile. You can send me an email using that.

> it should rename ITusHttpClientConfigre ？

That sounds good.